src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1338,7 +1357,11 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
 
   static closeAudioContext(): void {
     if (DefaultDeviceController.audioContext) {
-      DefaultDeviceController.audioContext.close();
+      try {
+        DefaultDeviceController.audioContext.close();
+      } catch (e) {
+        // Nothing we can do.
+      }
```

Comment:
This prevents failures if the context is somehow an `OfflineAudioContext`, which throws when you try to close it.